tests/testthat/test-along.R

```diff
@@ -0,0 +1,13 @@
+context("along")
+
+test_that("list_along works", {
+  x <- 1:5
+  expect_identical(list_along(x), vector("list", 5))
+})
+
+test_that("rep_along works", {
+  x1 <- 1:5
+  x2 <- 6:10
+  y <- 1:3
+  expect_identical(rep_along(x1, y), rep_along(x2, y))
```

Comment:
I think this would be a bit easier to take in at a glance if it used simpler inline data, i.e.

```R
expect_equal(rep_along(c("c", "b", "a"), 1:3)
```